data/mapcss-mapping.csv

```diff
@@ -1350,3 +1350,4 @@ recycling|cans;[recycling:cans=yes];;name;int_name;1351;
 recycling|shoes;[recycling:shoes=yes];;name;int_name;1352;
 recycling|green_waste;[recycling:green_waste=yes];;name;int_name;1353;
 recycling|cartons;[recycling:cartons=yes];;name;int_name;1354;
+cuisine=bubble_tea;1335
```

Comment:
Instead of adding a new line there please replace existing but not used one (`deprecated`), e.g. there is a 1324 close to other cuisines.

Reply:
Fixed in https://github.com/organicmaps/organicmaps/pull/2107/commits/bc27e8dd60dd8f816f8d14a04bc3784bb3c68058